src/toolkits/drawing_classifier/drawing_classifier.cpp

```diff
@@ -279,10 +279,11 @@ void drawing_classifier::init_training(
   // Read user-specified options.
   init_options(opts);
 
-  if (read_state<flexible_type>("random_seed") == FLEX_UNDEFINED) {
```

Comment:
read_statue will through exception if key is not found instead of return UNDEFINED. Or I may miss something here. @shantanuchhabra

Reply:
added back. Just read the `init_options`.